lessons/Early-Math/distributive-property/langs/en-us/index.adoc

```diff
@@ -0,0 +1,150 @@
+= The Distributive Property
+
+@description{Students develop an understanding of the Distributive Property by studying equivalent Circles of Evaluation. They then apply their understanding of the Distributive Property in a variety of contexts.}
+
+@lesson-prereqs{identity-property}
+
+@keywords{Circle of Evaluation, }
+
+[@lesson-intro-table]
+|===
+
+| Lesson Goals
+| Students will be able to...
+
+* Recognize how distribution results in equivalent expressions.
+* Apply the Distributive Property to expressions with numbers and variables.
+* Use distribution to efficiently complete mental computations.
+
+
+| Student-facing Lesson Goals
+|
+
+* Let's explore the Distributive Property and Circles of Evaluation.
+
+| Materials
+|[.materials-links]
+@material-links
+
+| Key Points For The Facilitator
+|
+* A common approach to teaching the Distributive Property is by using an area model. This approach works well for students who are comfortable visualizing and computing area - but may leave other students confused.
+* We introduce the Distributive Property by inviting students to look at patterns in _structure_, and then later apply what they know using the area model.
+
+|===
+
+== The Distributive Property @duration{30 minutes}
+
+=== Overview
+
+Students develop an understanding of the Distributive Property by looking at equivalent Circles.
+
+=== Launch
+
+Elise and Dani are discussing the two Circles, below.
+
+[.embedded, cols="^.^3,^.^3", grid="none", stripes="none" frame="none"]
+|===
+| @show{(coe '(* 9 (+ 40 6)))} | @show{(coe '(+ (* 9 40) (* 9 6)))}
+|===
+
+Elise thinks that the Circles are equivalent. She says, "I see 9, 40, and 6 in both Circles. The Circles include the same values, and even the same operations, so they must be equivalent."
+
+Dani disagrees. She says, "I don't think so... The Circle on the right has multiplication at the top, but the Circle on the right has addition on the top. And why does 9 show up twice in the Circle on the right?"
+
+[.lesson-instruction]
+--
+- Do you agree with Elise? Dani? Why?
+** _Allow students to share their approaches. The Circles are equivalent - but Elise's explanation is not complete._
+- Try using computation to evaluate each Circle. What do you notice?
+** _Sample responses: The Circles are equivalent. They both evaluate to 414. To evaluate the first Circle, I found a *product*. To evaluate the second Circle, I found a *sum*._
+--
+
+Share with students that the Circles are equivalent because of the @vocab{Distributive Property}, whereby @math{a \times (b + c) = ab + ac}. Facilitate a discussion where students think deeply about what is happening in the scenario above, namely, that distribution allows us to rewrite a product as a sum or a sum as a product.
+
+[.lesson-point]
+Multiplying the sum of two addends by a number produces the same result as multiplying _each_ addend by that number and then adding the products.
+
```

Comment:
I wonder whether this explanation might be more digestible without the word addend?

> Multiplying the sum of two numbers by a third numbers produces the same result as multiplying each of the numbers by the third number before finding the sum.